README.rst

```diff
@@ -63,6 +63,9 @@ What's New?
 in development
```

Comment:
Very minor comment... Maybe remove this for the release to have a cleaner readme on pypi.

https://pypi.org/project/pydoctor/#what-s-new